importer_client/python/timesketch_import_client/importer.py

```diff
@@ -174,6 +177,21 @@ def _fix_data_frame(self, data_frame):
             if 'timestamp' in data_frame:
                 data_frame['datetime'] = data_frame['timestamp'].dt.strftime(
```

Comment:
You wanna make sure that timestamp is an int here. I encountered this when hacking on the CLI client and all of a sudden the timestamp was a str and this raised an error.

`data_frame['timestamp'].astype(int).dt.strftime ... `

Reply:
done